src/ontology/mondo-edit.obo

```diff
@@ -435886,6 +435902,14 @@ xref: RO:0002573
 id: has_onset
 name: has onset
 
+[Typedef]:
+id: has_onset_before
```

Comment:
there shouldn't be colons after [Typedef]

Reply:
I fixed this :)

Reply:
thx!